### DISCUSSION TITLE
Specifying folders to sync

### DISCUSSION THREAD

**Discussion by dazinator** [abraunegg/onedrive#1281] [Q&A] [answered]
I'd like to set an allow list of the folders to sync - for example if I add a new folder to onedrive, I don't want this new folder to be synced automatically, unless I explicitly "allow" it to be by updating the config.

I have read the FAQ and there was nothing there about this. I have read the Usage Guide specifically on the Skip Dir option. It seemed to me from that that the Skip Dir option only acts as a block list - i.e use this setting to match folders that you don't want to sync. I am not sure whether I can use it to get the behaviour I desire - where I only explicitly want to sync specific folders and nothing else.

Many thanks for anyone who can help with this question.

**Comment by abraunegg** [comment, marked as answer]
@dazinator 
The only way to acheive what you want to do us to utilise / configure 'sync_list' - refer to https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#selective-sync-via-sync_list-file

**Reply by dazinator** [reply]
Ah sorry I neglected to see that option. I will use that mechanism- thank you very much.
